lambdas/test-lambda/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id "java"
+    id "jacoco"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation "com.amazonaws:aws-lambda-java-core:$rootProject.ext.dependencyVersions.awsLambdaJavaCore"
```

Comment:
If we're using a shared versioning, I'd be keen to apply it to all deps, not just this one